mmv1/third_party/terraform/.teamcity/components/constants.kt

```diff
@@ -19,8 +20,8 @@ const val DefaultParallelism = 6
 // this is updated semi-regularly
 const val DefaultTerraformCoreVersion = "1.11.0"
 
-// This represents a cron view of days of the week
-const val DefaultDaysOfWeek = "*"
+// This represents a cron view of days of the week - we skip saturday to allow release diff tests to run
+const val DefaultDaysOfWeek = "1-6"
```

Comment:
I think we should leave this off for now, ie. revert these lines. To start out, we'll just be experimenting with a low number of tests, and we don't want to have to handle the missing day of regular tests quite yet (particularly regarding metrics).

Reply:
just reverted in latest commit